active/InformationBar/InformationBar.md

```diff
@@ -497,6 +555,34 @@ unsealed runtimeclass InfoBar : Windows.UI.Xaml.Controls.ContentControl
 }
 ```
 
+## Microsoft.UI.Xaml.Controls.Primitives
+
+```cs
+[webhosthidden]
+unsealed runtimeclass InfoBarPanel : Windows.UI.Xaml.Controls.Panel
+{
+    InfoBarPanel();
+
+    // PaddingInHorizontalOrientation property
+    Windows.UI.Xaml.Thickness PaddingInHorizontalOrientation;
```

Comment:
[Suggestion](https://github.com/microsoft/microsoft-ui-xaml/issues/3701#issuecomment-735314353) from @robloo: 

PaddingInHorizontalOrientation => HorizontalOrientationPadding

Reply:
And I did mean that for all cases. I think there are around 4 properties total doing that.

Reply:
Recommend:
     HorizontalOrientationPadding 
     VerticalOrientationPadding 
     InfoBarPanel.HorizontalOrientationMargin (attached)
     InfoBarPanel.VerticalOrientationMargin (attached)